apps/db/lib/db/resource.ex

```diff
@@ -196,6 +199,17 @@ defmodule DB.Resource do
   @spec validate(__MODULE__.t()) :: {:error, any} | {:ok, map()}
   def validate(%__MODULE__{url: nil}), do: {:error, "No url"}
 
+  def validate(%__MODULE__{url: url, format: "gbfs"}) do
+    {:ok,
+     %{
+       "metadata" =>
+         Transport.Shared.GBFSMetadata.Wrapper.compute_feed_metadata(
+           url,
+           "https://#{System.get_env("DOMAIN_NAME", "")}"
```

Comment:
Ça c'est bof, dites-moi si vous avez une meilleure idée. J'ai besoin de l'URL de base de l'app web, pour valider si le CORS autorise le PAN. Je peux pas utiliser `TransportWeb.Endpoint.url()`, donc j'utilise la variable d'env qui contient le domaine, mais du coup je hardcode le protocole.

Ptet que savoir si on a du CORS n'est plus nécessaire maintenant qu'on a du GeoJSON maison pour les GBFS https://github.com/etalab/transport-site/pull/1968

Reply:
Si on laisse tomber l'umbrella bientot, ça ne me dérange pas d'avoir un truc légèrement bancal. Par contre, on évite d'utiliser le System.get_env ailleurs que dans la config. En général on utilise plutot le Application.get_env() dans le code après avoir pris la valeur une seule fois avec System.get_env dans la config.

Aussi je vois que jusqu'à présent on avait une valeur par défaut pour domaine name (`host: System.get_env("DOMAIN_NAME") || "transport.data.gouv.fr", port: 443]`). Du coup si la variable disparait, ça risque de se comporter bizarrement.

ils en parlent un peu là : https://elixirforum.com/t/environment-variables-not-updating-in-release/41862/2